src/server/main_service.cc

```diff
@@ -1146,9 +1146,10 @@ void Service::DispatchCommand(ArgSlice args, SinkReplyBuilder* builder,
   DCHECK(!args.empty());
   DCHECK_NE(0u, shard_set->size()) << "Init was not called";
 
-  absl::Cleanup clear_last_error([builder]() { builder->ConsumeLastError(); });
   ServerState& etl = *ServerState::tlocal();
 
+  builder->ResetLastError();
```

Comment:
Why do we need this here ? What's wrong with cleaning it up via `absl::Cleanup` ? We call this here before we invoke the command, we call this again via `InvokeCmd`. It seems redundant